.env

```diff
@@ -0,0 +1,2 @@
+USER = andreyshulga95
+PASSWORD = 2Eqlsa9xcN90
```

Comment:
now i can use your creds, thanks :)

Reply:
you mean i need to add a file .env in .gitignore?

Reply:
yes